emulators/qemu-guest-agent/src/opnsense/scripts/OPNsense/QemuGuestAgent/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Check if the kernel module exists
+KERNMOD='virtio_console'
+if [ -e /boot/kernel/${KERNMOD}.ko ]; then
+    # Load module
+    kldload $KERNMOD 2>&1
```

Comment:
Might already be loaded. I think kldstat -m can test for this?

Reply:
Yes, in theory, but it does not work for whatever reason...

```
root@opnsense: # kldstat 
Id Refs Address                Size Name
 1   35 0xffffffff80200000  26904e8 kernel
 2    1 0xffffffff82891000    10250 carp.ko
 3    1 0xffffffff828a2000     f998 if_bridge.ko
 4    2 0xffffffff828b2000     72a8 bridgestp.ko
 5    1 0xffffffff828ba000     3e78 if_enc.ko
 6    1 0xffffffff828be000     b1c0 if_gre.ko
 7    1 0xffffffff828ca000    16008 if_lagg.ko
 8    1 0xffffffff828e1000     8b60 if_tap.ko
 9    3 0xffffffff828ea000    583b8 pf.ko
10    1 0xffffffff82943000     2af8 pflog.ko
11    1 0xffffffff82946000     ebd0 pfsync.ko
12    1 0xffffffff82a12000     2668 intpm.ko
13    1 0xffffffff82a15000      b50 smbus.ko
14    1 0xffffffff82a16000     18a0 uhid.ko
15    1 0xffffffff82a18000     1aa0 wmt.ko
16    1 0xffffffff82a1a000     2aac virtio_console.ko

root@opnsense: # kldstat -m virtio_console
kldstat: can't find module virtio_console: No such file or directory

root@opnsense: # kldload virtio_console.ko
kldload: can't load virtio_console.ko: module already loaded or in kernel
```

...but it works for other modules:

```
root@opnsense: # kldstat -m carp
Id  Refs Name
  1    1 carp
```

Reply:
I could still do something like `if ! kldstat | grep virtio_console; then  kldload virtio_console; fi`, but that's not more efficient, so nothing to gain here I guess.

Reply:
Weird, but alright then 😊